variables.tf

```diff
@@ -1,37 +1,37 @@
-variable aws_region {
+variable "aws_region" {
   description = "AWS region"
   type        = string
   default     = "us-east-2"
```

Comment:
The region changes from us-east-1 to east-2 between the intial configuration branch and this one. Is that intentional?

https://github.com/hashicorp/learn-terraform-count-foreach/blob/foreach-initial-configuration/variables.tf#L4

https://github.com/hashicorp/learn-terraform-count-foreach/blob/learn-terraform-foreach/variables.tf#L4